lib/fluent/plugin/in_udp.rb

```diff
@@ -22,12 +22,35 @@ class UdpInput < SocketUtil::BaseInput
 
     config_set_default :port, 5160
     config_param :body_size_limit, :size, default: 4096
+    config_param :delimiter, :string, :default => ""
 
     def listen(callback)
       log.info "listening udp socket on #{@bind}:#{@port}"
       @usock = SocketUtil.create_udp_socket(@bind)
       @usock.bind(@bind, @port)
+      @has_delimiter = @delimiter != ''
       SocketUtil::UdpHandler.new(@usock, log, @body_size_limit, callback)
     end
+
+    def on_message(msg, addr)
+      return super(msg, addr) unless @has_delimiter
```

Comment:
> Some developers tend to write down default values demonstratively in config files for some reasons; as documentation, or any from their painful experiences, etc.

```
<source>
  @type udp
  delimiter ""
</source>
```
If the delimiter set to an empty string, how `in_udp` should perform? Which behaviour will be approved for developers most?
1. no splitting
2. split by each character